quickstart/daml/licensing/daml/Licensing/License.daml

```diff
@@ -43,6 +51,19 @@ template LicenseRenewalRequest with
   where 
     signatory user, provider
 
+    choice LicenseRenewalRequest_Cancellation : LicenseRenewalCancellation
+      with  
+        actor : Party -- Party that initiated the renewal request.
+        reasons  : Metadata
+      controller 
+        actor
+      do
+        now <- getTime
+        pure LicenseRenewalCancellation with
+          actor
+          reasons
+          cancellationTime = now
```

Comment:
Not required, as the ledger-effective time is available on the transaction itself.

Reply:
See: https://github.com/DACH-NY/canton/blob/82080478ad637030007e5424d29cb9090066fd19/community/ledger-api/src/main/protobuf/com/daml/ledger/api/v2/transaction.proto#L51